### PR TITLE
Documentation simply 4173

### PR DESCRIPTION
**Pull request by thinkwebengineer**
## Description

Add documentation for endpoints listed in Simply 4173.

## Motivation and Context

Improve api documentation and utilization.

## Checklist:

- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: api/documentation/controller.py [hunk 1]
```diff
@@ -738,6 +760,127 @@ def addSchemes(self):
...
+                'book': {
+                    'type': 'object',
+                    'properties': {
+                        'identifier_type': {'type': 'string'},
+                        'identifier': {'type': 'string'}
+                    }
+                },
+                'complaints': {
+                    'type': 'object',
+                    'description': 'A counter object of complaint types for a given work'
```
Comment (mwbenowitz) [inline api/documentation/controller.py]: Is it possible what the counter object looks like? Or is that not obvious from the codebase? Knowing what kind of types of complaints can be here would be helpful.

File: api/admin/routes.py [hunk 2]
```diff
@@ -906,6 +950,57 @@ def refresh(identifier_type, identifier):
...
+          schema:
+            type: string
+          description: An identifier for a work record
+      requestBody:
+        required: true
+        content:
+          multipart/form-data:
+            schema: 
+              type: string
+              name: type
```
Comment (mwbenowitz) [inline api/admin/routes.py:984]: What's required to go here? If not comprehensive an example would be really helpful.

File: api/admin/routes.py [hunk 3]
```diff
@@ -1127,7 +1287,37 @@ def suppressed():
 @returns_json_or_response_or_problem_detail
 @requires_admin
 def genres():
-    """Returns a JSON representation of complete genre tree."""
+    """Returns a JSON representation of complete genre tree.
+    ---
+    get:
+      tags: 
+        - works
```
Comment (mwbenowitz) [inline api/admin/routes.py]: Should probably be moved to the `standard lists` section we created in for SIMPLY-4165. Moving it there now will cause a PR conflict I think so we'll just have to resolve that when it comes time to merge.

File: api/admin/routes.py [hunk 4]
```diff
@@ -1257,23 +1447,79 @@ def circulation_events():
 @requires_admin
 def stats():
     """Return object of library statisticss by library name and total statistics.
+    ---
+    get:
+      tags:
+        - analytics
+      summary: Return object of library statistics by library name and total stattistics.
```
Comment (mwbenowitz) [inline api/admin/routes.py]: nit: typo in 2nd statistics!

**Review by mwbenowitz — COMMENTED** [review]
A few minor things but 99% of the way there!

File: api/admin/routes.py (hunk 3 above)
Comment (thinkwebengineer) [reply api/admin/routes.py]: Merged develop into this branch to bring all commits up to date.

**Review by mwbenowitz — APPROVED** [review]
👍

**Pull request merged**